packages/cli/README.md

```diff
@@ -45,7 +57,7 @@ USAGE
 
 OPTIONS
   -c, --ceramic=ceramic  Ceramic API URL
-  -k, --key=key    DID Key
+  -k, --key=key          DID Private Key
```

Comment:
Assuming the glaze config get/set commands are for setting values in the local config file (like how the equivalent commands in the ceramic cli works today), then I assume they don't need or use the --key arg.